apps/src/music/player/ProgramSequencer.js

```diff
@@ -0,0 +1,106 @@
+// The ProgramSequencer supports the play_together and play_sequential blocks
```

Comment:
Very cool! I like this direction of moving the logic of when to play stuff out of the block generator code.